### ISSUE TITLE
Analog sticks on Gamepad doesn't work - Jet Set Radio can't be played

### ISSUE THREAD

**Issue by GoogleCodeExporter** [aloone/nulldc#98]
```
I'm sorry if there is an easy solution and I'm wasting time, but I reckon 
if anyone can answer it it's you guys. Noone else seems to be able to help.

I'm trying to play Jet Set Radio on May 13 Build (v1.0.4), and I've hooked 
up my controller with the PuruPuru input plugin. My gamepad has an analog 
stick, so I thought it would use analog controls instead of 1/0 style 
something.

But I can't finish of the bigger graffiti tags in the game, because it 
doesn't sense the analog movements. How do I calibrate nullDC to pick 'em 
up?

Should I use a better input plugin? I've heard about a "nullDC joystick/n" 
plugin, but I can't find it in my list. Do I have to calibrate the 
deadzone, whatever that is?

Thanks for advance
```

Original issue reported on code.google.com by `BenLaserlove` on 30 May 2010 at 10:42
- Merged into: #21

**Comment by GoogleCodeExporter** [comment]
```

Its already FIXED!!!! a long time ago!!!! Just Update it here
http://www.megaupload.com/?d=MPMKKYJ0
P.S.
And next time, first, you must learn to searching on google for example! And 
try read
forums! And only after that you must write a comment! Becouse the r22 exist on 
this
site very long time! 
Anyway keep your eyes open and more thinking! Good luck!


```

Original comment by `N2008eo...@rambler.ru` on 30 May 2010 at 11:55

**Comment by GoogleCodeExporter** [comment]
```
Always test with latest revisions please. Then go back and see if it worked 
before. =P

Now let's wait confirmation from the poster beforehand.
```

Original comment by `KrossX3` on 30 May 2010 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Ah, maybe that stuff worked. My cdi images seems to stop working now, because 
they're 
not NRG, nor MDF/MDS. No matter, I'll sort that stuff out.

Is there a way to get the new revisions without having to compile them from the 
source? 
Have no clue about how to do that stuff.

Also, you're implying that I didn't look for this stuff. :P I just thought the 
first 
one on the download page would work, because, uh, stable release or something.
```

Original comment by `BenLaserlove` on 1 Jun 2010 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
No 'stable' release yet. =P

Get a recent one from here:
http://www.mediafire.com/krossx

And please do test the issue.
```

Original comment by `KrossX3` on 1 Jun 2010 at 6:42

**Comment by GoogleCodeExporter** [comment]
```
Okay, it works fine now. Thanks for putting up with me. =)
```

Original comment by `BenLaserlove` on 1 Jun 2010 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
Roger.


```

Original comment by `KrossX3` on 2 Jun 2010 at 3:57
- Changed state: **Duplicate**
